src/components/rn-bottom-drawer/Animator.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react';
+import {
+  PanResponder,
+  Animated,
+  Dimensions,
+  StyleSheet
+} from 'react-native';
+
+const DOWN_STATE = 0
+const UP_STATE = 1
+const SCREEN_HEIGHT = Dimensions.get('window').height;
+const SCREEN_WIDTH = Dimensions.get('window').width;
+
+export default class Animator extends Component {
+  constructor(props) {
+    super(props);
+
+    this.position = new Animated.ValueXY(this.props.currentPosition);
+
+    this._panResponder = PanResponder.create({
+      onStartShouldSetPanResponder: () => true,
+      onPanResponderMove: this._handlePanResponderMove,
+      onPanResponderRelease: this._handlePanResponderRelease
+    });
+  }
+
+  componentWillReceiveProps(nextProps) {
```

Comment:
This method is deprecated as far as I know, I believe the new one is `componentDidReceiveProps`